### PR TITLE
Prints each result type conveniently (raw,int,str,bool)

### PR DESCRIPTION
**Pull request by nickazg**
**What current issue(s) does this address?, or what feature is it adding?**
Smart contract results are always printed as a bytearray, But its nice to be able what the output represents as different types. eg its currently difficult to see the value of a BigInterger, while a bool is easy to tell, its also handy to see it as a quick True/False. 

**How did you solve this problem?**
Because neo-python is awesome there was already conversion methods provided with the ByteArray object. All I did was duplicate and indent the existing "results" print statement

**How did you make sure your solution works?**
I have been using it for a while, and I am using preexisting methods and just simply printing them.

**Did you add any tests?**
no, I am using preexisting methods and just simply printing them

**Are there any special changes in the code that we should be aware of?**
Not really, everything else is the same

### CODE REVIEW

**Comment by metachris** [comment]
Oh, you could just have edited the PR -- at the top you can click the pencil and change the target branch. Nvm. There is still a conflict with `prompt.py` though.

**Comment by metachris** [comment]
I'm afraid the conflict is not really resolved -- take a look at the changes here: https://github.com/CityOfZion/neo-python/pull/196/files

**Comment by nickazg** [comment]
Yeah I forgot to update my fork to the latest so figured it was easier starting a new request.. That all looks good now?

**Comment by nickazg** [comment]
Ahh annoying, my local repo isnt complaining when I pull from here.. Bare with me for a sec haha.

**Comment by nickazg** [comment]
yeah my local prompt file wasn't updating in my IDE when I pulled, reopening gave me the conflict. Should be all good now, sorry!

File: prompt.py [hunk 1]
```diff
@@ -736,10 +736,14 @@ def load_smart_contract(self, args):
...
-                    print("Results %s " % [str(item) for item in results])
-                    print("Deploy Invoke TX GAS cost: %s" % (tx.Gas.value / Fixed8.D))
-                    print("Deploy Invoke TX fee: %s" % (fee.value / Fixed8.D))
+                    print("Total operations executed: %s " % num_ops)
+                    print("Results:")
+                    print("\t(Raw) %s" % [str(item) for item in results])
+                    print("\t(Int) %s" % [item.GetBigInteger() for item in results])
+                    print("\t(Str) %s" % [item.GetString() for item in results])
+                    print("\t(Bool) %s" % [item.GetBoolean() for item in results])
+                    print("Deploy Invoke TX gas cost: %s " % (tx.Gas.value / Fixed8.D))
```
Comment (metachris) [inline prompt.py]: keep `GAS` all caps please

**Comment by metachris** [comment]
Interesting contribution, first of all thanks for taking the time 👍 

I'm not 100% sure we want this as default behavious, would like to hear the thoughts of @localhuman on this.

Alternatively, we might put this behind a feature-flag or setting (`config verbose-deploy on/off` and persist this in the userprefs), for instance.

**Comment by metachris** [comment]
Really not 100% sure what to do with this. I somehow like it, actually, but not sure it should be default. We can leave it open for now and think a little more about it. For new developers, this info is actually quite nice to have :)

**Comment by nickazg** [comment]
Yeah its only really a visual convenience thing. Especially for new developers who have so much to take in already, it can be somewhat confusing/unintuitive if they input an Int/bool/string and get returned a b'd' when they were expecting 100 (they should still understand why though) . If they haven't implemented their own node yet its just another step, even if you understand why a byte array is returned. Just food for thought.

**Pull request merged**